pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -73,6 +73,8 @@ const (
 	dockerPidFile         = "/var/run/docker.pid"
 	containerdProcessName = "docker-containerd"
 	containerdPidFile     = "/run/docker/libcontainerd/docker-containerd.pid"
+	crioProcessName       = "crio"
+	crioPidFile           = "/var/run/crio.pid"
```

Comment:
What do you think about using /var/run/crio/pid here?

Reply:
Either path is fine by me.

Reply:
Updated.